pkg/types/ack_adoption_reconciler.go

```diff
@@ -0,0 +1,19 @@
+// Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"). You may
+// not use this file except in compliance with the License. A copy of the
+// License is located at
+//
+//     http://aws.amazon.com/apache2.0/
+//
+// or in the "license" file accompanying this file. This file is distributed
+// on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+// express or implied. See the License for the specific language governing
+// permissions and limitations under the License.
+
+package types
+
+// ACKAdoptionReconciler
+type ACKAdoptionReconciler interface {
+	ACKReconciler
+}
```

Comment:
Is there a need for this interface?

Reply:
Just futureproofing. But not at this stage, no

Reply:
OK, let's leave it out for now, then.